script.js

```diff
@@ -126,23 +126,12 @@ function renderQuestion() {
   buttonElems[optionBIndex].innerText = question.charDiedAfter;
 }
 
-function endGame() {
-  console.log('game ended');
-  const clear1 = document.querySelectorAll('.js-options-list');
-  clear1.forEach(elem => {
-    elem.className='is-hidden1';
-  })
-  const clear2 = document.querySelectorAll('.js-quiz-question'); 
-  clear2.forEach(elem => {
-    elem.className='is-hidden1';
-  })
-  const gametext = document.querySelectorAll('.is-hidden2'); 
-  gametext.forEach(elem => {
-    elem.className='end-text';
-  })
+function renderGameOver() {
+  document.querySelector('.js-options-list').classList.add('is-hidden');
```

Comment:
although it's a good practise to use querySelectorAll and loop through all of them. Here I know there would be only one options-list and quiz-question so just querySelector is enough